packages/ui-react/src/components/Header/Header.tsx

```diff
@@ -215,7 +217,7 @@ const Header: React.FC<Props> = ({
         </div>
         {logoSrc && (
           <div className={styles.brand}>
-            <Logo src={logoSrc} onLoad={() => setLogoLoaded(true)} />
+            <Logo alt={siteName} src={logoSrc} onLoad={() => setLogoLoaded(true)} />
```

Comment:
IMHO, we purposely left the `alt` empty because we don't describe a meaningful description. If we take an example from the [stichting accessibility site](https://accessibility.nl), we should describe it as `Homepage of ${siteName}`.

Reply:
The alt text was "logo" previously instead of it being empty. I do believe making it empty is not a good solution either, because the image isn't decorative or has an adjacent text alternative.

But I do think prefixing it with "Homepage of.." is handy, because users can reasonably assume the screen reader is pronouncing the site logo.

When we're getting audited I suggest keeping our sitename short and concise.

Reply:
Changed.